masschallenge/django-accelerator
# [AC-9653]: Create models for Home Community

[AC-9653](https://masschallenge.atlassian.net/browse/AC-9653)

**How to test:*

Ensure definition of done is met from [AC-9653](https://masschallenge.atlassian.net/browse/AC-9653)

Siblings: https://github.com/masschallenge/accelerate/pull/3356/files
